diffusion/callbacks/log_diffusion_images.py

```diff
@@ -38,62 +40,67 @@ class LogDiffusionImages(Callback):
 
     def __init__(self,
                  prompts: List[str],
-                 size: Optional[int] = 256,
+                 size: Optional[Union[Tuple[int, int], int]] = 256,
```

Comment:
This shouldn't have the `Optional` I think. A few others like this too here, if you want to fix them.

Reply:
I removed optional from size and guidance_scale, then added a type to num_inference_steps. Let me know if others should be removed!